release-notes.md

```diff
@@ -1,5 +1,19 @@
 # Release notes
 
+## October 2, 2019
+
+The following updates were made in this release:
+
+**iOS Target 2.1.4**
+
+* Target session id and edge host will now be persisted and reset if there is no activity for the configured session timeout. If `target.sessionTimeout` is not configured, the default session timeout will be 30 minutes.
```

Comment:
- please replace "id" with "ID". 
- Is "edge host" a proper name? Should it be capitalized like "Edge Host"?

- Please reword this section so that it reads as follows:
"* If there is no activity for the configured session timeout, the Target session ID and Edge host are now persisted and reset. 
If `target.sessionTimeout` is not configured, the default session timeout is 30 minutes." (Move this sentence to the next line under the bullet.)